app/uk/gov/hmrc/gform/validation/ComponentValidator.scala

```diff
@@ -36,6 +36,45 @@ import uk.gov.hmrc.referencechecker.CorporationTaxReferenceChecker
 import scala.util.matching.Regex
 
 object ComponentValidator {
+  // format: off
+  val genericLongTextErrorPattern = "generic.longText.error.pattern"
```

Comment:
What effect this `// format: off` is supposed to have?

If you to want improve readability I'd vote for something like that:
```
  val genericLongTextErrorPattern                            = "generic.longText.error.pattern"
  val genericReferenceNumberErrorPattern                     = "generic.referenceNumber.error.pattern"
  val genericCrnErrorInvalid                                 = "generic.crn.error.invalid"
  val genericEoriErrorPattern                                = "generic.eori.error.pattern"
  val genericUkEoriErrorPattern                              = "generic.ukEori.error.pattern"
  val genericChildBenefitNumberErrorPattern                  = "generic.childBenefitNumber.error.pattern"
  val genericNonUKCountryCodeErrorPattern                    = "generic.nonUKCountryCode.error.pattern"
  val genericCountryCodeErrorPattern                         = "generic.countryCode.error.pattern"
  val genericErrorTelephoneNumber                            = "generic.error.telephoneNumber"
  val genericShortTextErrorPattern                           = "generic.shortText.error.pattern"
  val genericErrorLookup                                     = "generic.error.lookup"
  val genericErrorRegistry                                   = "generic.error.registry"
  val genericErrorRequired                                   = "generic.error.required"
  val genericErrorParentSubmissionRefSameAsFormSubmissionRef = "generic.error.parentSubmissionRefSameAsFormSubmissionRef"
  val genericErrorExactNumbers                               = "generic.error.exactNumbers"
  val genericErrorSubmissionRef                              = "generic.error.submissionRef"
  val genericErrorMaxWhole                                   = "generic.error.maxWhole"
  val genericErrorPositiveNumber                             = "generic.error.positiveNumber"
  val genericErrorMaxLengthNoDecimals                        = "generic.error.maxLength.noDecimals"
  val genericErrorMaxLengthMaxDecimals                       = "generic.error.maxLength.maxDecimals"
  val genericErrorWholeNumber                                = "generic.error.wholeNumber"
  val genericErrorMaxDecimals                                = "generic.error.maxDecimals"
  val genericErrorPositiveWholeNumber                        = "generic.error.positiveWholeNumber"
  val genericErrorNumber                                     = "generic.error.number"
  val genericErrorInvalid                                    = "generic.error.invalid"
  val genericErrorMaxLength                                  = "generic.error.maxLength"
  val genericErrorMinLength                                  = "generic.error.minLength"
  val genericVrnErrorPattern                                 = "generic.vrn.error.pattern"
  val genericGovernmentIdNotExist                            = "generic.governmentId.not.exist"
  val genericGovernmentIdErrorPattern                        = "generic.governmentId.error.pattern"
  val genericErrorEmail                                      = "generic.error.email"
  val choiceErrorRequired                                    = "choice.error.required"
  val timeErrorRequired                                      = "time.error.required"
```